### PR TITLE
added new component car-repair x2

### PR DESCRIPTION
**Pull request by Mazvys**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -3,12 +3,14 @@ import { Routes, RouterModule } from '@angular/router';
 import { HomeComponent } from './home/home.component';
 import { CarComponent } from './car/car.component';
 import { ReportComponent } from './report/report.component';
+import { CarRepairComponent } from './car-repair/car-repair.component';
 
 const routes: Routes = [
   {path:"", redirectTo:"home", pathMatch:'full'},
   {path:"home", component:HomeComponent},
```
Comment (MerlinX-Angular) [inline src/app/app-routing.module.ts:10]: here problem

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -3,12 +3,14 @@ import { Routes, RouterModule } from '@angular/router';
 import { HomeComponent } from './home/home.component';
 import { CarComponent } from './car/car.component';
 import { ReportComponent } from './report/report.component';
+import { CarRepairComponent } from './car-repair/car-repair.component';
```
Comment (MerlinX-Angular) [inline src/app/app-routing.module.ts:6]: hh

**Review by MerlinX-Angular — CHANGES_REQUESTED** [review]

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (Mazvys) [reply src/app/app-routing.module.ts:10]: everything is good here

**Review by MerlinX-Angular — APPROVED** [review]

**Review by MerlinX-Angular — APPROVED** [review]

**Comment by MerlinX-Angular** [comment]
all good

**Pull request merged**